internal/server/adapters/storage/storage.go

```diff
@@ -2,20 +2,33 @@ package storage
 
 import (
 	"errors"
+	"fmt"
 
+	"github.com/agatma/sprint1-http-server/internal/server/adapters/storage/file"
```

Comment:
Пора переименовать пакет:

```suggestion
	"github.com/agatma/metrics/internal/server/adapters/storage/file"
```